### ISSUE TITLE
Add support for cross-lingual voice cloning

### ISSUE THREAD

**Issue by guranu** [camenduru/OpenVoice-colab#2]
Hello, i was wondering if it's possible to add cross lingual voice cloning to the colab because there's only english and chinese support in the colab.

**Comment by iamjamilkhan** [comment]
Yes please make colab for multi language model as described in demo2 using own base speaker or openai api

**Comment by VitalyKaino** [comment]
> Да, пожалуйста, сделайте совместную работу для многоязычной модели, как описано в демонстрации 2, используя собственный базовый динамик или openai API.

Can you show an example? add French or German using the base speaker
